exchanges/poloniex/poloniex_types.go

```diff
@@ -351,10 +351,16 @@ type WebsocketTrollboxMessage struct {
 	Reputation    float64
 }
 
-// WsCommand defines the request params after a websocket connection has been
-// established
+type wsOp string
+
+const (
+	wsSubscribeOp   wsOp = "subscribe"
+	wsUnsubscribeOp wsOp = "unsubscribe"
+)
```

Comment:
Suggestion: Could be an exported type in subscriptions package another exchange has a similar thing.

Reply:
I looked hard at this again, and couldn't get onboard with stringifying it because it's a func param.
Your argument is most important though... these are exported structs. But they shouldn't be.
This probably isn't the fix you were looking for 😂  Not trying to be awkward, I promise 🤞 

Fixed 56e545a

Let me know if you'll let that fly.